docs/guide/yaml/test/example_yaml/simple-tomcat-app-sensor-test.yaml

```diff
@@ -0,0 +1,24 @@
+brooklyn.catalog:
+  id: simple-tomcat-app-sensor-test
+  version: 1.0
+  itemType: template
+  iconUrl: http://tomcat.apache.org/images/tomcat.png
+  name: Simple Tomcat Application Sensor Test
+  license: Apache-2.0
+  item:
+    brooklyn.config:
+      simple.confg: someValue
```

Comment:
Agreed, specifics of packaging up tests can be dealt with in the context of a test runner.